### DISCUSSION TITLE
why debian repository has old mydumper version?

### DISCUSSION THREAD

**Discussion by saifalicw** [mydumper/mydumper#1524] [General]
Hello,

In our company, we want to use this tool for backups and restores, but due to security complaints they are hesitant to install it using GitHub packages for Debian, we found the Debian repository is not up to date with new Mydumper versions, so if you could help us in these regards and push a new Debian version to the Debian repo we get latest release version.

Thanks

**Comment by davidducos** [comment]
Hi @saifalicw 
Because I don't have the knowledge on how to do it... I need someone to help me on #1464

**Reply by saifalicw** [reply]
Hey @davidducos the process is as simple as this:
https://wiki.debian.org/DebianRepository/SetupWithReprepro

I can offer some help but obviously the servers and all that need to be setup by you.

**Reply by davidducos** [reply]
We already implemented a repo, this is the official documentation https://mydumper.github.io/mydumper/docs/html/installing.html
